### ISSUE TITLE
Do someone have trained the ImageNet model reaching 57.x% testing during training?

### ISSUE THREAD

**Issue by dabilied** [BVLC/caffe#843]
1:I have run the ImageNet process twice; The first time I use "convert -resize 256x256! $name $name" to convert images;the second time I use "launch_resize_and_crop_images.sh" and get the 256*256 image in a crop way;However the accuracy during training all  stayed at about 54.x%,really having a gap.Have someone  reappear the "Caffe Reference ImageNet Model",could you kindly give me some details(my gmail is fengyanchao56@gmail.com);thank you!
2:I use python-predict the imagenet val-50000 images,with oversampling,on "Caffe Reference ImageNet Model",the accuracy seems higher than 57.x% as it has  data enhancement .So ,if we make a enhancement in the training process,the model should be better(Now it only center and mirror 1:2),why caffer don't execute this? As "ilvsrc12_train_leveldb"  converted by images is very big,data enhancement should not bigger this any more~~;
3:fine-turning is useful,could caffer provide some tutorial such as form imagenet classification model to detection model;how finetune_net.cpp works?It retains which parameters,and initialized others?
4:Now caffe can be regard as a systems for detection;However it must have matlab ,  code in c++ or python is more suitable  in application ; Can c++ format be available in the future? 
5:At "http://caffe.berkeleyvision.org/getting_pretrained_models.html" the pretrained model are (232.6M）and (243.9MB）,however they are all about 232.6M , 243.9MB may as you look it on linux and just divide 1000 . Is this right ?
6:How big is imagenet_train_leveldb ? I remeber it is about 200+G, I recreated it some days ago, it just 
 130+G How could this been ? imagenet_val_leveldb is always 9.2G
